src/widgets/places/ui/places/places.module.scss

```diff
@@ -0,0 +1,24 @@
+.places {
+  white-space: nowrap;
+
+  &.content {
```

Comment:
I thought you asked to remove `& .card-content` -> `.card-content` in all ishiu's

Reply:
seems to be resolved https://github.com/rolling-scopes/site/pull/661#discussion_r1876810571